libuavcan/test/transport/dispatcher.cpp

```diff
@@ -76,29 +73,25 @@ TEST(Dispatcher, Reception)
     /*
      * Test environment
      */
-    static const uavcan::DataTypeDescriptor TYPES[4] =
-    {
-        makeDataType(uavcan::DataTypeKindMessage, 1),
-        makeDataType(uavcan::DataTypeKindMessage, 2),
-        makeDataType(uavcan::DataTypeKindService, 1),
-        makeDataType(uavcan::DataTypeKindService, 1)
-    };
+    static const uavcan::DataTypeDescriptor TYPES[4] = {
+        makeDataType(uavcan::DataTypeKindMessage, 1), makeDataType(uavcan::DataTypeKindMessage, 2),
+        makeDataType(uavcan::DataTypeKindService, 1), makeDataType(uavcan::DataTypeKindService, 1)};
```

Comment:
Misplaced braces; also, the entries should not have been rearranged. This is important, the original formatting should have been retained.